### PR TITLE
Issue #5934: Move default layouts into their own table

### LINKED ISSUE #5934: [UX] Create a separate section for the 2 default layouts in the layout management page
This is a follow-up to #5077 and related to #4871.

The `default` and `admin_default` layouts are a special case: fallback layouts that are used when no other layout applies. However, we are listing them along with the rest of the layouts that belong in the "Layout overrides" section.

I think that these deserve a special place, especially since:
- they are already being rendered last in the list - regardless of weight/alpha
- they already have a separate "group description": `Default layouts (used on all paths without a specific layout)`

I'm basically suggesting to split this subsection of the layout overrides table into it's own table:
image

### PR DESCRIPTION
**Pull request by klonos**
Fixes https://github.com/backdrop/backdrop-issues/issues/5934

### CODE REVIEW

File: core/modules/layout/layout.admin.inc [hunk 1]
```diff
@@ -16,92 +16,87 @@ function layout_list_page() {
...
+        $row[] = '';
+      }
     }
-
+    // Add the heading row to the non-default layout path groups.
     if (isset($page_callbacks[$path]) && $page_callbacks[$path] == 'layout_page_callback') {
       $stand_alone_rows[] = array('data' => $row, 'class' => array('layout-group'));
     }
-    else {
+    elseif ($path !=='') {
```
Comment (bugfolder) [inline core/modules/layout/layout.admin.inc]:
```suggestion
    elseif ($path !== '') {
```
Need a space.

File: core/modules/layout/layout.admin.inc [hunk 2]
```diff
@@ -201,8 +200,8 @@ function layout_list_page() {
     '#attributes' => array('class' => array('layout-list')),
     '#empty' => t('No layout pages have been created yet.'),
     '#weight' => 1,
-  );
-  $page['override'] = array(
+   );
+   $page['override'] = array(
```
Comment (klonos) [inline core/modules/layout/layout.admin.inc]:
```suggestion
  );
  $page['override'] = array(
```

File: core/modules/layout/layout.admin.inc [hunk 3]
```diff
@@ -211,10 +210,22 @@ function layout_list_page() {
...
+    '#attributes' => array('class' => array('layout-list')),
     '#empty' => t('No layout overrides have been created yet.'),
     '#weight' => 3,
   );
+  $page['default'] = array(
+    '#type' => 'help',
+    '#markup' => '<h2>' . t('Default layouts') . '</h2>' . t('Used as a fallback for all paths without a matching layout.'),
+    '#weight' => 4,
+   );
+   $page['default_table'] = array(
```
Comment (klonos) [inline core/modules/layout/layout.admin.inc]:
```suggestion
  );
  $page['default_table'] = array(
```

**Comment by quicksketch** [comment]
Merged into 1.x and 1.26.x. Thanks @klonos!

**Pull request merged**